xml/System.Text.RegularExpressions/RegexParseError.xml

```diff
@@ -34,7 +42,8 @@
       </ReturnValue>
       <MemberValue>17</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>An alternation construct in a regular expression contains a comment.</summary>
+        <remarks>To be added.</remarks>
```

Comment:
Why are all the `<remarks>` set to `To be added.`?

Reply:
I believe `To be added` is the default for `remarks`. IIRC `remarks` don't show up in Intellisense.

Reply:
What about docs.microsoft.com?

Reply:
I asked @carlossanlop exactly that question a while back. Waiting for an answer :)

Reply:
If "To be added" is found, then nothing is shown. It's the default empty string for MS Docs.

The reason why it was added is because the DocsPortingTool expects the xml element to exist. I could get it fixed, so that if it's not found, I won't add it unless there's something to port.

```suggestion
```

Reply:
@eerhardt @pgovind I removed them.